ledger/complete/wal/encoding.go

```diff
@@ -61,6 +61,7 @@ func Decode(data []byte) (operation WALOperation, rootHash ledger.RootHash, upda
 	switch operation {
 	case WALUpdate:
 		update, err = ledger.DecodeTrieUpdate(data[1:])
+		rootHash = update.RootHash
```

Comment:
This fixes a bug, that the root hash is not returned when decoding a record in the wal file.